SoftLayer/CLI/routes.py

```diff
@@ -47,6 +47,7 @@
     ('config', 'SoftLayer.CLI.config'),
     ('config:setup', 'SoftLayer.CLI.config.setup:cli'),
     ('config:show', 'SoftLayer.CLI.config.show:cli'),
```

Comment:
since "sl conflict setup" and "sl setup" do the same thing .. either we should deprecate one (whichever one sounds less intuitive) or we should document that these 2 commands do the same thing ..else it can leave the users confused